### PR TITLE
Feature/spp885/add logging to export lambda

### PR DESCRIPTION
**Pull request by rhiGriff**
# Description
Added spp-logger to dbexport lambda
https://collaborate2.ons.gov.uk/jira/browse/SPP-885 
Upgraded go to v 1.16 and added go.mod and go.sum files

# How to test?
In environment 1271

Send a message on the spp-es-takeon-egestion-spp1271-db-export-input queue

Example message:
{
	"snapshot_id": "14e0fb27-d450-44d4-8452-9f6996b00e27",
	"surveyperiods": [
	  {
		"survey": "023",
		"period": "201904"
	  },
	  {
		"survey": "023",
		"period": "201903"
	  },
	  {
		"survey": "066",
		"period": "201903"
	  },
	  {
		"survey": "067",
		"period": "201904"
	  }
	]
  }

Check cloudwatch logs.

Also please change log level to Debug and check

### CODE REVIEW

**Review by fayaz0211 — APPROVED** [review]
Tested the changes by sending the payload to the export queue and can able to see the export lambda logs for debug level and also verified the snapshot being upload to the export s3 bucket

File: takeon-dbexport/main.go [hunk 1]
```diff
@@ -40,43 +49,66 @@ type OutputMessage struct {
 	Successful bool   `json:"successful"`
 }
 
+// LoggerContextStruct ...
+type LoggerContextStruct struct {
```
Comment (ranjit-das) [inline takeon-dbexport/main.go:53]: Looks like LoggerContextStruct is not used anywhere. So can it be removed?

File: takeon-dbexport/main.go [hunk 2]
```diff
@@ -40,43 +49,66 @@ type OutputMessage struct {
...
 	lambda.Start(handle)
 
 }
 
 func handle(ctx context.Context, sqsEvent events.SQSEvent) error {
-	fmt.Println("Starting the application...")
+	env := strings.Split(lambdaName, "-")
+
+	loggerConfig = spp_logger.Config{
+		Service:     "Validation",
```
Comment (ranjit-das) [inline takeon-dbexport/main.go:69]: Should Service be any other name e.g. Egestion or any other name that Dave can confirm?

File: takeon-dbexport/main.go [hunk 3]
```diff
@@ -184,7 +214,7 @@ func sendToSqs(snapshotid string, filename string, successful bool) {
 
 	DataToSend, err := json.Marshal(outputMessage)
 	if err != nil {
-		fmt.Printf("An error occured while marshaling DataToSend: %s", err)
+		logger.Error("An error occured while marshaling DataToSend: ", err)
 	}
 	fmt.Printf("DataToSend %v\n", string(DataToSend))
```
Comment (ranjit-das) [inline takeon-dbexport/main.go:219]: This Printf should be changed to logger.info/debug

**Review by ranjit-das — CHANGES_REQUESTED** [review]

File: takeon-dbexport/main.go (hunk 1 above)
Comment (rhiGriff) [reply takeon-dbexport/main.go:53]: Good spot thank you

**Review by ranjit-das — APPROVED** [review]
Tested and found as expected

**Pull request merged**